Source/Core/DolphinQt/Config/LogWidget.cpp

```diff
@@ -8,29 +8,32 @@
 #include <QComboBox>
 #include <QFont>
 #include <QFontDatabase>
-#include <QGroupBox>
+#include <QGridLayout>
+#include <QPlainTextEdit>
 #include <QPushButton>
-#include <QScrollBar>
-#include <QTextEdit>
 #include <QTimer>
-#include <QVBoxLayout>
-
-#include "Common/FileUtil.h"
-#include "Common/StringUtil.h"
 
 #include "Core/ConfigManager.h"
 
-#include "DolphinQt/QtUtils/QueueOnObject.h"
 #include "DolphinQt/Settings.h"
 
 // Delay in ms between calls of UpdateLog()
 constexpr int UPDATE_LOG_DELAY = 100;
 // Maximum lines to process at a time
-constexpr int MAX_LOG_LINES = 200;
+constexpr int MAX_LOG_LINES_TO_UPDATE = 200;
+// Maximum number of lines to show in log viewer
+constexpr int MAX_LOG_LINES = 5000;
```

Comment:
If you make this of type `size_t`, you don't need the specialization of `std::min()` on line 77